saleor/settings.py

```diff
@@ -13,7 +14,14 @@ def get_list(text):
     return [item.strip() for item in text.split(',')]
 
 
-DEBUG = ast.literal_eval(os.environ.get('DEBUG', 'True'))
+def get_bool(name, default_value):
```

Comment:
I wonder if we can find more descriptive name for this. `get_env_variable_value`? I'd look for something not too long, but providing enough information.

Reply:
`get_bool_from_env` maybe?

Reply:
Yes, sounds better. Shall we add an `try` that warns if someone set the env variables incorrect i.e. `DEBUG="tru"`?